core/node/da_dispatcher/src/da_dispatcher.rs

```diff
@@ -101,6 +104,7 @@ impl DataAvailabilityDispatcher {
                 .insert_l1_batch_da(
                     batch.l1_batch_number,
                     dispatch_response.blob_id.as_str(),
+                    self.config.enable_onchain_verification,
```

Comment:
Why do we insert this information to the database?
If it corresponds to the L1 config (e.g. "starting from the batch X, the settings change"), then it doesn't feel robust enough -- e.g. there are no guarantees that the value on the contract will change syncrhonously.
If it doesn't correspond to the L1 config (e.g. operator can do whatever), then we can just look at `self.config` in `poll_inclusion_data`.

Rephrasing,
- If it's possible to toggle state on the contract, I'd expect this code to be more resilient.
- Otherwise, it can be simplified.

Reply:
that's a very good point
I initially decided to keep it in the DB to make the migration easier (in theory you can define whether the batches get DA verification via DB), but just thought about it and:
- if migrating from `dontVerify` to `verify` -> the verification will be done for every batch after the server restarts with the new config, which makes sense
- if migrating from `verify` to `dontVerify` -> it doesn't matter what the old config was if the L1 can accept empty DA inclusion proofs

So I went with the simplification, and I will get back to this when working on the check to L1 in that another PR